app/controllers/register/trustees/individual/TrusteeAddressController.scala

```diff
@@ -67,9 +64,9 @@ class TrusteeAddressController @Inject()(
             ManualAddressViewModel(
               postCall(mode, Index(index), srn),
               countryOptions.options,
-              title = Message(title),
-              heading = Message(heading),
-              hint = Some(Message(hint)),
+              title = Message("messages__trustee__individual__address__confirm__title"),
+              heading = Message("messages__trustee__individual__address__confirm__heading", details.fullName),
+              hint = Some(Message("messages__trustee__individual__address__confirm__lede")),
               secondaryHeader = Some(details.fullName),
```

Comment:
secondary header is no longer used.

Reply:
Great spot! I'm removing it from the view model also since it isn't being used by the view